src/pages/RFOX/hooks/useGetUnstakingRequestCountQuery.ts

```diff
@@ -35,14 +35,14 @@ export const useGetUnstakingRequestCountQuery = <SelectData = UnstakingRequestCo
   const queryKey: GetUnstakingRequestCountQueryKey = useMemo(
     () => [
       'readContract',
-      {
+      serialize({
```

Comment:
shouldn't be needed but probably a good practice to use wherever we do wagmi wrappers